transcom/mymove
# B-21040 Prime can view third addresses

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21040)
## [INT PR](https://github.com/transcom/mymove/pull/13645)

## Summary

The first part of the effort to allow prime to view/update/create moves with second and third addresses. This work allows prime to view second and third addresses, and updates our prime simulator in a similar fashion. After discussions with PO, this does not require a feature flag.

Setup:
Make a move available to prime with shipments of each type, each with second/third addresses where possible.

To test:
1. Login to the prime simulator and access the move you setup. Confirm that the move details page has the correct data for second/third addresses.
2. Access the moves, and confirm that the shipment details page has accurate data for second/third addresses (for PPMs,  use the network tab to confirm the data is being delivered as part of the shipment details; third addresses display/update on prime sim page will be handled in the update story)

[ANDI SCREENSHOTS](https://www13.v1host.com/USTRANSCOM38/Default.aspx?Page=Widgets%2fDetails%2fAttachmentDetails&AssetWindow=true&AssetContext=Attachment%3a1007986)
